src/main/java/com/gw2auth/oauth2/server/service/gw2account/apitoken/Gw2AccountApiTokenServiceImpl.java

```diff
@@ -65,7 +65,7 @@ public void updateApiTokensValid(Instant lastValidCheckTime, Collection<Gw2Accou
         this.gw2AccountApiTokenRepository.updateApiTokensValid(lastValidCheckTime, updates);
     }
 
-    @Scheduled(fixedRate = 10L, timeUnit = TimeUnit.MINUTES)
+//    @Scheduled(fixedRate = 10L, timeUnit = TimeUnit.MINUTES)
```

Comment:
this should not be removed

Reply:
Good that you spotted this. I missed this after doing some testing.